apps/sample-app/src/components/Calendar.tsx

```diff
@@ -1,21 +1,39 @@
 import './styles.css';
 
 import { Button } from '@fluentui/react-components';
-import { calendar } from '@microsoft/teams-js';
+import { app, calendar } from '@microsoft/teams-js';
 import React from 'react';
 
-export const Calendar: React.FunctionComponent = () => {
-  const handleCalendar = async (): Promise<void> => {
-    // create calendar paramaters
+const teamsDeepLinkHost = 'teams.microsoft.com';
+const teamsDeepLinkProtocol = 'https';
+const teamsDeepLinkAttendeesUrlParameterName = 'attendees';
+const teamsDeepLinkUrlPathForCalendar = '/l/meeting/new';
+
+export const handleNewMeeting = async (): Promise<void> => {
+  if (!calendar.isSupported()) {
+    const calendarParams: calendar.ComposeMeetingParams = {
+      attendees: ['emailAddress@microsoft.com'],
```

Comment:
Since you have this in two places (here and on line 26) you should just define it once as a constant outside and then reference it in both places.